sdk/nodejs/storage/zMixins.ts

```diff
@@ -261,6 +261,32 @@ export interface QueueContext extends appservice.Context<void> {
     };
 }
 
+/**
+ * Host settings specific to the Storage Queue plugin.
+ *
+ * For more details see https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-queue#host-json
+ */
+export interface QueuesHostSettings extends appservice.HostSettings {

```

Comment:
Why 'Queues' instead of 'Queue'?

Reply:
Because the extension is called `queues`... But then I broke this rule with Topic naming. I'll rename to 'Queue'.